resources/views/auth/emails/password.blade.php

```diff
@@ -1,7 +1,7 @@
-Hi!
+Hi!<br/>
 <br/>
 You are receiving this email because we received a password reset request from you. If you requested this, please click here to reset your password: <a href="{{ $link = url('password/reset', $token).'?email='.urlencode($user->getEmailForPasswordReset()) }}"> {{ $link }} </a><br/>
 <br/>
-If you did not request a password reset, you can ignore this email. Your password will not change and your account is safe.
+If you did not request a password reset, you can ignore this email. Your password will not change and your account is safe.<br/>
```

Comment:
this line outputting more of what we wanted (see screenshot) so mimicking this on other lines